prawns/evm-contracts/src/processors/moonbeamProcessor.ts

```diff
@@ -0,0 +1,23 @@
+import { SubstrateProcessor } from '@subsquid/substrate-processor';
+import ethereumExecutedHanndler from '../handlers/ethereum.Executed';
+import { SubstrateNetwork } from '../model';
+import types from './moonbeamTypesBundle';
+
+const processor = new SubstrateProcessor(
+  'litentry_squid_evm_contracts_moonbeam'
+);
+
+processor.setTypesBundle(types);
+processor.setBatchSize(100);
+// processor.setBlockRange({ from: 209450 });
```

Comment:
Was this forgotten here?

Reply:
yes but I've realised I need it to look into an erc1155 issue